lib/trace.c

```diff
@@ -941,11 +941,11 @@ M0_INTERNAL void m0_trace_buf_header_init(struct m0_trace_buf_header *tbh, size_
 	m0_atomic64_set(&tbh->tbh_rec_cnt, 0);
 
 	strncpy(tbh->tbh_motr_version, bi->bi_version_string,
-		sizeof tbh->tbh_motr_version);
+		sizeof tbh->tbh_motr_version - 1);
```

Comment:
In this case, do we need to manually add '\0' to the end of dest string "tbh->tbh_motr_version"?

Reply:
The "tbh" header itself is initialized to be all zero. So, no need to manually add the terminating '\0'.

Reply:
Yes.